Securing Governance Through Revamped MOBILE Staking

```diff
@@ -0,0 +1,59 @@
+
+2. **Emissions Apportionment:**
+   - Allocate 4% of total MOBILE emissions to veMOBILE stakers, creating a direct and proportional reward system for participants.
+
+3. **Reduced Staking Minimums:**
+   - Establish a minimum stake duration of 1 day with a 1x vote power multiplier.
+   - Implement a daily multiplier increment of 0.0025 for every day a stake remains active. This progressive approach aligns with sustained commitment, rewarding long-term participants.
+
+4. **Unlocking Mechanism:**
+   - Stakers can unlock at any time and will be subject to a 14 epoch (approximately equal to 14 days) cool down period. During the cool down period, the tokens that were locked for veMOBILE will remain locked, but the veMOBILE will not have any voting power or receive any emissions. Stakers can elect to unlock all or some portion of their stake at any time.
+
+5. **Voting Power Increase:**
+    - A future HIP can introduce an additional multiplier increase of for every vote cast by veMOBILE holders in the governance process. This HIP will include the development work to make that possible; however, it will be implemented with an increase set at 0.00 unless another HIP is proposed and passed before implementation. It is the expectation of the authors that the other HIP will be discussed in parallel with or shortly after the passage of this HIP.
+
+## Rationale
+
+### Strengthening Network Security
+
+- The dynamic staking mechanism should create an incentive for more veMOBILE to be locked, reducing the risk of network attacks and enhancing overall security.
+
+### Fostering Long-Term Commitment
+
+- The daily multiplier increment encourages veMOBILE holders to commit to longer staking periods, aligning interests and promoting sustained engagement.
+
+### Enhancing Governance Participation
+
+- By rewarding veMOBILE holders with an additional multiplier for casting votes, this proposal aims to increase participation in the governance process, fortifying the network against potential governance attacks.
+
+### Improving Liquidity Management For Stakers
+
+- The unlocking mechanism provides a balanced approach to liquidity management, allowing veMOBILE holders access to their tokens without long term commitments of up to 4 years in the current system. We believe a more dynamic approach to lock ups will make reduce the barrier for people looking to lock up their tokens. Additionally, we feel, given the current data for veMOBILE locks as well as other veHNT/veDNT tokens that people often lock fewer tokens for longer durations under the current system. The network prefers more tokens locked than fewer.
+
+## Stakeholders
+
+- All current and future veMOBILE holders as well as MOBILE holders.
+
```

Comment:
```suggestion
- Authors: [Max Gold](https://github.com/maxgold91), [Nik Hawks](https://github.com/gristlekinginc), [Keith Rettig](https://github.com/KeithRettig), Hans, JD, Jess M, [Jay M](https://github.com/jaym2518), [Andy Zyvoloski](https://github.com/heatedlime)
- Start Date: 2024-06-11
- Category: Economic, Governance, Technical
- Original HIP PR: [Link to Original HIP PR]
- Tracking Issue: [Link to Original HIP PR]
- Vote Requirements: veMOBILE Holders

---

## Summary

This HIP aims to update the veMOBILE voting mechanism, creating an incentive for network participants to lock their MOBILE tokens, increasing voter participation. The proposal introduces a multifaceted approach, including unlocking currently locked tokens, a daily multiplier increment, and creates the mechanism for a subsequent HIP to add voting power boost for active participation in governance.

## Motivation

TODO

## Stakeholders

All current and future veMOBILE holders as well as MOBILE holders.

## Detailed Explanation

1. **Current veMOBILE unlock:**
    - Upon implementation, all MOBILE currently locked in veMOBILE at the time of implementation will be able to unlock their tokens without any cool down period for the first 7 epochs after implementation. Any legacy veMOBILE will instantly reduce to a 1x on the implementation date and be treated like all other veMOBILE, aside from the 7 epoch amnesty period described above.

2. **Emissions Apportionment:**
    - Allocate 4% of total MOBILE emissions to veMOBILE stakers, creating a direct and proportional reward system for participants.

3. **Reduced Staking Minimums:**
    - Establish a minimum stake duration of 1 day with a 1x vote power multiplier.
    - Implement a daily multiplier increment of 0.0025 for every day a stake remains active. This progressive approach aligns with sustained commitment, rewarding long-term participants.

4. **Unlocking Mechanism:**
    - Stakers can unlock at any time and will be subject to a 14 epoch (approximately equal to 14 days) cool down period. During the cool down period, the tokens that were locked for veMOBILE will remain locked, but the veMOBILE will not have any voting power or receive any emissions. Stakers can elect to unlock all or some portion of their stake at any time.

5. **Voting Power Increase:**
    - A future HIP can introduce an additional multiplier increase of for every vote cast by veMOBILE holders in the governance process. This HIP will include the development work to make that possible; however, it will be implemented with an increase set at 0.00 unless another HIP is proposed and passed before implementation. It is the expectation of the authors that the other HIP will be discussed in parallel with or shortly after the passage of this HIP.

## Drawbacks

`TODO`

## Rationale and Alternatives

### Strengthening Network Security

- The dynamic staking mechanism should create an incentive for more veMOBILE to be locked, reducing the risk of network attacks and enhancing overall security.

### Fostering Long-Term Commitment

- The daily multiplier increment encourages veMOBILE holders to commit to longer staking periods, aligning interests and promoting sustained engagement.

### Enhancing Governance Participation

- By rewarding veMOBILE holders with an additional multiplier for casting votes, this proposal aims to increase participation in the governance process, fortifying the network against potential governance attacks.

### Improving Liquidity Management For Stakers

- The unlocking mechanism provides a balanced approach to liquidity management, allowing veMOBILE holders access to their tokens without long term commitments of up to 4 years in the current system. We believe a more dynamic approach to lock ups will make reduce the barrier for people looking to lock up their tokens. Additionally, we feel, given the current data for veMOBILE locks as well as other veHNT/veDNT tokens that people often lock fewer tokens for longer durations under the current system. The network prefers more tokens locked than fewer.

## Unresolved Questions

`TODO`

## Deployment Impact

`TODO`

### Documentation Updates

`TODO`

### User Impact

`TODO`

```

Reply:
reformatted the entire HIP to match with HIP standards.